components/script/dom/htmlscriptelement.rs

```diff
@@ -148,9 +231,22 @@ pub enum ScriptType {
     Module,
 }
 
+#[derive(JSTraceable, MallocSizeOf)]
+pub struct CompiledSourceCode {
+    #[ignore_malloc_size_of = "SM handles JS values"]
+    pub source_code: Heap<*mut JSScript>,
+    pub script: DOMString,
```

Comment:
I think it would be clearer if we call this `original_text`.

Reply:
I'll do that change

Reply:
I just pushed the changes to this branch, but not showing here. That's weird